.github/workflows/publish-central.yml

```diff
@@ -15,7 +15,7 @@ jobs:
           ref: "refs/tags/${{ github.event.inputs.tag }}"
       - uses: actions/setup-java@v3
         with:
-          java-version: 11
+          java-version: 17
```

Comment:
this requires you to set a bunch of additional env vars (due to a bug in the nexus staging plugin).

Reply:
(only for the maven central build)

Reply:
Those options?

https://github.com/cryptomator/fuse-nio-adapter/blob/afea456431739e3d1fd07f544f97a2c5cf01abbc/.github/workflows/publish-central.yml#L31-L35

Reply:
exactly

Reply:
Done in https://github.com/cryptomator/dokany-nio-adapter/pull/53/commits/2680a9531d286a15efcc295c02fc7995af38884b